2.evaluate_data/README.md

```diff
@@ -0,0 +1,18 @@
+# Apply phenotypic profiling model to JUMP data
+
+In this module, we generate single-cell probabilities for each of the 15 phenotypic classes by applying the [phenotypic profiling model](https://github.com/WayScience/phenotypic_profiling_model).
+There are two model types, final and shuffled baseline, where single-cell features are randomized independently. 
+We output one probability `parquet` file that contains all of the single-cell data per plate and well that contains all relevant metadata.
```

Comment:
```suggestion
We output one file per plate that contains phenotypic probabilities and relevant metadata for all of the single-cells.
The files we output are in `parquet` format.
```

Reply:
For clarity

Reply:
@MattsonCam Can you clarify if the "predict_sc_probabilities.ipynb" notebook outputs the model predictions as one file with all plates or a file per plate. Based on what I can see, it looks like the first option:

```python
        # Store the prediction dataframes
        model_preds.append(predsdf)

# Concatenate the model predictions
model_preds = pd.concat(model_preds)

# Save predictions
model_preds.to_parquet(f"{output_proba_path}/model_probabilities.parquet", index=True)
```
Once this is clarified I can for sure make this change.